docs/what.rst

```diff
@@ -0,0 +1,29 @@
+=====================
+What is a Certificate?
+=====================
+
+A public key or digital *certificate* (formerly called an SSL certificate) uses a public key 
+and a private key to enable secure communication between a client program (web browser, email client, 
+etc.) and a server over an encrypted SSL (secure socket layer) or TLS (transport layer security) connection.
+The certificate is used both to encrypt the communication and to identify the server. The certificate
+includes information about the key, information about the server identity, and the digital signature
+of the certificate issuer. If the issuer is trusted by the software that initiates the communication,
+and the signature is valid, then the key can be used to communicate securely with the server identified by 
+the certificate. Using a certificate is a good way to prevent "man-in-the-middle" attacks, in which
+someone in between you and the server you think you are talking to is able to insert their own (harmful)
+content.
+
+You can use Certbot to easily obtain and configure a free certificate from Let's Encrypt, a
+joint project of EFF, Mozilla, and many other sponsors.
+
+Certificates and Lineages
+=========================
+
+Certbot introduces the concept of a *lineage,* which is a collection of all the versions of a certificate
+plus Certbot configuration information maintained for that certificate from
+renewal to renewal. Whenever you renew a certificate, Certbot keeps the same configuration unless
+you explicitly change it. If you add domains, you can either add them to an existing lineage or create
+a new one. 
+
+See also:
+:ref:`updating_certs`
```

Comment:
That's a nicely detailed and well formulated description. May I add a few comments and suggestions? (I am using **bold** font to indicate added and edited portions of the text.

**Web** browsers and other software commonly require a certificate when making a secure connection to an Internet service. ...

[Comment: There are also image browsers, file browsers, etc. that are working locally.]

The certificate contains a public key. The certificate's recipient—the operator of the site or service described by the certificate—also has the corresponding private key, which needs to be kept secret. The certificate is used to identify the server when establishing a secure connection **and to encrypt the communication for this connection**. 

> The main security benefit of using a certificate is the prevention of "man-in-the-middle" attacks, in which someone in between you and the server you think you are talking to is able to spy on you or insert their own (harmful) content.

That's not quite right. That's only one security benefit. (And I wouldn't call it a "benefit", rather it is the purpose.)
But since the encryption of the secure channel relies on the key (certificate), having a certificate/key is a necessity.  Hence, I suggest something like this:

The certificate contains a public key. The certificate's recipient—the operator of the site or service described by the certificate—also has the corresponding private key, which needs to be kept secret. **There are two roles that a certificate plays in a secure connection:  to identify the server when establishing the connection and to encrypt the communication for the connection**. The certificate includes information about the public key, information about the server identity, and the digital signature of the certificate authority that issued the certificate. If the issuer is trusted by the software that initiates the communication, and the signature is valid, then the certificate confirms that the public key can be used to communicate securely with the server identified by the certificate. **One of the purposes of identifying the server is to prevent** "man-in-the-middle" attacks, in which someone **in-between** you and the server you think you are talking to is able to spy on you or insert their own (harmful) content.

[Note the dash in *in-between*, between the words ;-)  Ref: https://www.merriam-webster.com/dictionary/in-between ]

Would it make sense listing all components of the certificate explicitly and together -- somewhere around here?


> Certbot introduces the concept of a lineage, which is a collection of all the versions of a certificate and its associated files from renewal to renewal, plus Certbot configuration information for that certificate. 

What are the "associated files" that are neither parts of the certficate, not Certbot configuration information? Do you mean CSRs? Whatever you mean by them, why not list all components of a lineage explicitly?


> You can view a _list of lineages_, and their names, by running certbot _certificates_; for most operations with Certbot, you can _specify which lineage_ you want to act on using _--cert-name_.

I understand that you correctly described how Certbot works, but doesn't this sentence expose some awkwardness of the terminology used by Certbot? A naive question: if I want to view a list of *lineages* why don't I use a command `certbot lineages`? And if I want to specify which *lineage*, why don't I use --lineage-name? ;-)
I am not suggesting to introduce these commands; the existing commands are more intuitive. Rather, I am demonstrating why the notion of "lineage" is confusing. 

Actually, I'd suggest rephrasing this sentence to avoid this awkwardness as follows:

You can view a list of **certificate** lineages, and their names, by running `certbot certificates`. For most operations with Certbot, you can specify which **certificate** ~~lineage~~ you want to act on using `--cert-name`.

I think certbot is primarily acting on a certificate, not on a lineage. This is consistent with your description of the lineage and with certbot's commands:   you request, obtain, renew, modify, remove  a **certificate**, not a _lineage_. (You modify the lineage by modifying [replacing] the current certificate) You delete the _lineage_. (I've never done that yet, but I assume the entire lineage is deleted, right?)     And `certbot --help` output talks about acting on **certificates**, not on lineages.